fuzz/fsfuzzer.py.data/fsfuzzer.yaml

```diff
@@ -0,0 +1,4 @@
+setup:
+ filesystem : !mux
+  default : 
+   fstype : cramfs|ext2|ext3|ext4|swap|iso9660|msdos|vfat|xfs
```

Comment:
This is not supported fstype. I only pointed out what are the supported ones. What I had in mind is:

```
!mux
cramfs:
    fstype: cramfs
ext2:
    fstype: ext2
ext3:
    fstype: ext3
ext4:
    fstype: ext4
swap:
    fstype: swap
iso9660:
    fstype: iso9660
msdos:
    fstype: msdos
vfat:
    fstype: vfat
xfs:
    fstype: xfs
```

Which when executed would run all types as separate avocado tests.